VERSION

```diff
@@ -0,0 +1 @@
+1.0.0-pre1
```

Comment:
Yes, we should update the **Tilezen API** version in the new [VERSION](VERSION) file. 

Agree we should capture the **service** version external to the vector-datasource repo. Should that go into a new repo and/or one of the chef recipes? 

(I've updated the deployment checklist wiki, but it'll need to be updated again per outcome of above.)